vmngclient/api/partition_manager_api.py

```diff
@@ -48,21 +48,21 @@ def _set_default_partition(self, payload_devices: List[dict]) -> str:
 
     def set_current_partition_as_default(self, devices: List[Device]) -> str:
         """
-        Method to set current software version as default version
+        Set current software version as default version
 
         Args:
-            devices (List[Device]): For those devices default partition
-            going to be set
+            devices (List[Device]): For those devices default partition is
+                going to be set
 
         Returns:
-            str: action id
+            str: set default partition action id
         """
-        devs = self.device_versions.get_devices_current_version(devices)
-        return self._set_default_partition(devs)
+        devices_current_versions = self.device_versions.get_devices_current_version(devices)
```

Comment:
it's API goldening . Do we really need it ?

Reply:
I changed it, from now we have one common method for set default partition.